packages/tests/wdio.conf.js

```diff
@@ -221,8 +231,39 @@ exports.config = {
     /**
      * Function to be executed before a test (in Mocha/Jasmine) starts.
      */
-    // beforeTest: function (test, context) {
-    // },
+    beforeTest: function (test, context) {
+        // Override console to allow for STDOUT and file logging with one command
+        originalConsole = console
+
+        let filename = test['fullName'].replace(/\s/gi, '_');
+        let file = `${logsPath}/${test['start']}-${filename}.log`;
+
+        const logger = Logger.new(file);
+
+        console = (function(originalConsole){
```

Comment:
again mostly a stylistic comment that could be picked up by a linter, but we normally prefer arrow functions over function declarations